.gitignore

```diff
@@ -1 +1,2 @@
 /target
+Cargo.lock
```

Comment:
Any reason to add this ignore? Rust binaries usually commit the Cargo.lock file (I didn't yet, but probably will when I tag the first release).

https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

Reply:
No, I am completely indifferent in this regard and since I never developed software with Rust, I also didn't know that to do with the `Cargo.lock` file really.  

I just noticed that the header read  

```
# This file is automatically @generated by Cargo.
# It is not intended for manual editing.
```

and thus presumed there would be no use to use version control on that file. Probably, I should just not have added my .gitignore in the first place ;-) 

This is clearly not what this PR is all about...

Reply:
> Very cool. I can see that this could be useful. Let me see if I can follow what's going on.

 I am happy to answer questions regarding the Dockerfile and Github Action. 
 
Once the software is available on Bioconda, their CI will also take care of building the Docker images, so it is anyway rather temporary. Ultimately, putting it on Bioconda is even preferred by us, because at the NGI we exclusively run nf-core pipelines. So in order to use your tool in production, we would anyway write a nf-core module around it, which only works reasonably well, if a tool is available on Bioconda. 
 
> Happy to see interest in `rumidup` and `gensum`. I'm sure improvements are required for wider adoption. Especially regarding error reporting, so please submit an issue if you encounter anything odd or missing.

Yes, I will do. So far, we haven't had any issues with it.